packages/lodestar-cli/src/cmds/init/index.ts

```diff
@@ -1,11 +1,11 @@
-import {CommandModule} from "yargs";
+import {ICliCommand, ICliCommandOptions} from "../../util";
 import {IBeaconOptions, beaconOptions} from "../beacon/options";
-import {initHandler} from "./init";
+import {initHandler} from "./handler";
 import {IGlobalArgs} from "../../options";
 
-export const init: CommandModule<IGlobalArgs, IBeaconOptions> = {
+export const init: ICliCommand<IBeaconOptions, IGlobalArgs> = {
```

Comment:
Since you renamed `IValidatorCLIOptions` to `IValidatorCLIArgs`, should also rename `IBeaconOptions` to `IBeaconArgs` and any other

Reply:
I was unsure about that because in `packages/lodestar` it's called `IBeaconOptions`, won't it create confusion?

Reply:
I thought that was `IBeaconNodeOptions` in packages/lodestar

Reply:
Oh, right! No problem then for `IBeaconArgs`